Userland/Libraries/LibWeb/Painting/SVGPaintable.cpp

```diff
@@ -31,4 +33,48 @@ CSSPixelRect SVGPaintable::compute_absolute_rect() const
     return PaintableBox::compute_absolute_rect();
 }
 
+Optional<CSSPixelRect> SVGPaintable::get_masking_area() const
+{
+    if (!is<SVG::SVGGraphicsElement>(dom_node()))
+        return {};
```

Comment:
Would it make sense to put this function in `SVGGraphicsPaintable` instead?

Reply:
Yep :+1: